bugFix: moving scroll-padding-top prop to reference.css

wyankush opened the pull request:
This is a fix for issue #8080 

As mentioned on the issue:
 - Removed `scroll-padding-top` property from [main.css](https://github.com/Leaflet/Leaflet/blob/main/docs/docs/css/main.css)
 - Added it to [reference.css](https://github.com/Leaflet/Leaflet/blob/main/docs/docs/css/reference.css) file under `html` tag.

Malvoz commented:
Thanks @wyankush. I didn't fully investigate before opening the bug report. https://github.com/Leaflet/Leaflet/pull/7703/files shows that `overflow: hidden` was also introduced for sticky headings, can you also move that property?

Malvoz commented:
These changes will also need to be made to the same files under `docs/SlavaUkraini/docs/css`.

wyankush commented:
@Malvoz I can surely make the additional change.

I have one question though:
I wanted to know should I create a new pull request with the new change and delete this pull request?
I am new to contributing to open source projects(in fact this is my first contribution) so I am not exactly aware on how to proceed here.

Falke-Design commented:
@wyankush you can simply push a new commit to the same branch and the PR is automatically updated.

wyankush commented:
added the changes:

- removed `html` tag from [main.css](https://github.com/Leaflet/Leaflet/blob/main/docs/docs/css/main.css)

- added `overflow` property to  [reference.css](https://github.com/Leaflet/Leaflet/blob/main/docs/docs/css/reference.css)

Malvoz reviewed (changes requested):
Changes required to the same files in the `SlavaUkraini` folder: https://github.com/Leaflet/Leaflet/pull/8083#issuecomment-1076286052

wyankush commented:
Done the chances in `SlavaUkraini` folder

Malvoz commented on docs/docs/css/reference.css:
```diff
@@ -1,3 +1,8 @@
+html {
+	scroll-padding-top: 5em;
+    overflow: auto;
```
And finally, please indent `overflow: auto`.

wyankush replied:
fixed! Actually, `scroll-padding-top` had extra spacing.
